src/controller/CHIPDeviceController.cpp

```diff
@@ -3183,6 +3196,23 @@ void DeviceCommissioner::PerformCommissioningStep(DeviceProxy * proxy, Commissio
     }
     break;
     case CommissioningStage::kSendComplete: {
+        if (params.GetICDStayActiveDurationMsec().HasValue())
+        {
+            IcdManagement::Commands::StayActiveRequest::Type request;
+
+            request.stayActiveDuration = params.GetICDStayActiveDurationMsec().Value();
+            ChipLogError(Controller, "Send ICD StayActive with Duration %u", request.stayActiveDuration);
+            CHIP_ERROR err =
+                SendCommissioningCommand(proxy, request, OnICDManagementStayActiveResponse, OnBasicFailure, endpoint, timeout);
```

Comment:
You can't do two SendCommissioningCommand that are both part of the commissioning flow in parallel; that will lead to VerifyOrDie failures, no?

Please resolve merge conflicts with master and retest.  I am pretty sure this code crashes if it runs.  We really do want a separate commissioning step here, I suspect.

Reply:
agree. Just put back the kICDStayActive state between kFindOperational and kSendComplete, retest with and without ICDStayActiveDuration, thanks